model-engine/setup.cfg

```diff
@@ -11,15 +11,19 @@ omit =
     model_engine_server/core/celery/app.py
     model_engine_server/core/docker/ecr.py
     model_engine_server/db/base.py
+    model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py
```

Comment:
would ideally like to avoid having to skip code coverage for use cases, since we're actually able to test them (unlike a lot of the infra/ files)